### ISSUE TITLE
Bundles loader - feature request

### ISSUE THREAD

**Issue by MrElectronic** [vuejs/vue#2373]
Hi

I'm working on a Vue loader inspired by requirejs. Maybe some of you could use the idea.

Link to the github repo where I try to create a vue-bundle loader
https://github.com/your-budget-today/personal/tree/gh-pages

Bundles are partial html files where the goal is that each bundle file should contain css, template, routes and view-model. 

It is a work in progress, not production ready at the moment. 

Br.
Rune

**Comment by tomcon** [comment]
Looks good - interesting stuff

**Comment by yyx990803** [comment]
Very cool! This is something I've had in mind for a while - it's great for prototyping and small applications. Ideally, I'd love to see it support standard `*.vue` files outlined [here](http://vuejs.github.io/vue-loader/start/spec.html), excluding pre-processors. That way you can reuse the same `*.vue` files if you later want to transition to a webpack/browserify based setup.

I'm going to close this since this doesn't seem like an issue - feel free to share this on the forum's show & tell section.

**Comment by MrElectronic** [comment]
The general idea was to use it for prototyping. I don't know how good the performance will be in large apps. I have tried to change it so it would be able to mix single component files (without wrapping) and multi component files (with <vue-component> wrapping). Have never worked with Babel so will take a look at it and see if I can make it work in my script. Right now the script section must be a standard object.

**Comment by tomcon** [comment]
@MrElectronic Rune, I've made very small mods and am using this right now. My app became too large so now I have 8 components in .html files. The sections are standard .vue files (which I see you've taken Evan's advice too). Also, without webpack, as you know the html extension makes them good to work with in editors + intellisense. Its performance seems really fine! 

FYI I spent a couple of days getting into webpack but it just slowed me down totally. I'm not using ED2015 yet. Then I came across this: 
http://javascriptplayground.com/blog/2016/02/the-react-webpack-tooling-problem

and just used these 2 package.json lines to good effect 
"babel": "babel src --out-file public/js/bundle.js --watch",
"live-server": "live-server --port=3004"

Then found your file and am really productive again.
Will switch over to ES2015 & webpack in time, just not right now. 
Also don't think it is ideal for Evan to have one type of code in main vuejs.org and ES2015 on the sub-projects - doesn't help people learning imo. Fantastic work from him though!
